src/routes/transactions/transactions.service.ts

```diff
@@ -490,12 +489,12 @@ export class TransactionsService {
   private getFirstTransactionNonce(
     page: Page<DomainMultisigTransaction>,
   ): number | null {
-    return head(page.results)?.nonce ?? null;
+    return page.results[0]?.nonce ?? null;
```

Comment:
```suggestion
    return page.results?.[0]?.nonce ?? null;
```

I think we need to cover the case where `page.results` is `null` with optional chaining before accessing the array.